pages/docs/connect/webhook-events.mdx

```diff
@@ -0,0 +1,66 @@
+
+| Attribute  | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| \_links    | An object that contains relevant links to resources in the Dwolla API. Possible links can include: `self`, `account`, `resource`, and `external-party`. <br /> `self` - a link to the unique event <br /> `account` - a link to the Dwolla account that the application belongs to <br /> `resource` - a link to the resource that changed states. Used to lookup additional details returned on the resource itself <br /> `external-party` - a link to the external-party the Event belongs to |
+| id         | Unique Event ID. An Event ID is used along with the created timestamp for idempotent event processing.                                                                                                                                                                                                                                                                                                                                                                                           |
+| created    | ISO-8601 timestamp of when the event was created.                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| topic      | Type of action that occurred within Dwolla. <br/> **Format:** `<resource>.<related_resource>:<type/action>` or `<resource>:<type/action>`. <br/> For example, the event topic for when a funding source belonging to an external party is removed is `external_party.funding_source:created`.                                                                                                                                                                                                    |
```

Comment:
@ShreyaThapa should we change this example to `account.transfer:created` since that's likely the most known?

Reply:
We can! My intention was to have an example that showed the casing using in the topic so developers can know what to expect. Perhaps we can add more examples like so? :

```suggestion
| topic      | Type of action that occurred within Dwolla. <br/> **Format:** `<resource>.<related_resource>:<type/action>` or `<resource>:<type/action>`. <br/> For example, the event topic for when a transfer is created is `account.transfer:created`. <br/> Other examples include: `account:funding_source:removed`, `external_party:created`.     
```

Reply:
Yeah, that sounds good. Small tweak to change the first colon ":" in the account.funding_source:removed topic name and it should be g2g

Reply:
Oops! Good catch! I edited it in the suggestion.